examples/research_projects/rag/finetune_rag.py

```diff
@@ -495,18 +494,10 @@ def add_retriever_specific_args(parser):
             "for the distributed retriever. Has no effect when "
             "distributed_retriever is set to pytorch.",
         )
+        return parser
 
     @staticmethod
     def add_ray_specific_args(parser):
-        parser.add_argument(
```

Comment:
why delete this argument?

Reply:
As explained above, this argument is duplicate (See [here](https://github.com/huggingface/transformers/blob/8217d4e37fce48490a68af7e8ce902af16318132/examples/research_projects/rag/finetune_rag.py#L490-L508)) and causes an error

Reply:
I would remove the other one and keep this one instead; sincice it's a ray-specific arg

Reply:
Thank you for the comment @lhoestq 
I did so in the latest commit.